js/ui/diagram/ui.diagram.js

```diff
@@ -287,7 +288,26 @@ class Diagram extends Widget {
              * @type Enums.DiagramAutoLayout
              * @default 0
              */
-            layout: "tree"
+            layout: "tree",
+
+            /**
+             * @name dxDiagramOptions.export
+             * @type object
+             */
+            "export": {
+                /**
+                 * @name dxDiagramOptions.export.fileName
+                 * @type string
+                 * @default "PivotGrid"
```

Comment:
Default value is PivotGrid. Are you sure?

Reply:
mistake, thanks.